epsagon/events/dbapi.py

```diff
@@ -88,13 +87,16 @@ def __init__(
         # using 'WITH' clause
         operation = query.split()[0].lower()
```

Comment:
is it possible that query is empty so that split is of length 0 and query.split()[0] would fail?